pom.xml

```diff
@@ -93,12 +92,12 @@
       <dependency>
         <groupId>org.eclipse.sisu</groupId>
         <artifactId>org.eclipse.sisu.inject</artifactId>
-        <version>${sisu.version}</version>
+        <version>${version.sisu-maven-plugin}</version>
```

Comment:
the property name doesn't look related to such library.
I know it's coming from the parent but this is really confusing.

Reply:
Yes, is ugly, will figure out something. My point was really to not have "disconnected" the sisu version in this plugin vs parent.

Reply:
Isn't this inherited anyway?

Reply:
Huh, seems it is, removing whole depMgt section and moving guice version to dep, and that's did it.